core/services/twilio/src/main/java/com/mongodb/stitch/core/services/twilio/TwilioService.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2018-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb.stitch.core.services.twilio;
+
+public final class TwilioService {
```

Comment:
[q] Where is this used?

Reply:
I was thinking the admin SDK, but maybe this is better removed!